packages/react/src/components/Tree/TreeItem.tsx

```diff
@@ -43,10 +44,10 @@ export interface TreeItemProps extends UIComponentProps, ChildrenComponentProps
   exclusive?: boolean
 
   /** The index of the item among its sibbling */
-  index: number
+  index?: number
 
   /** Array of props for sub tree. */
-  items?: ShorthandValue[]
+  items?: ShorthandCollection<TreeItemProps>
```

Comment:
This should be `TreeProps`

Reply:
But then it won't be Collection.. Not sure if it should be changed, but it seems confusing as it is used in the `Tree.create(items, ..)` factory

Reply:
As `Tree` has `mappedArrayProp` it should be:
```suggestion
  items?: ShorthandValue<TreeProps> | ShorthandCollection<TreeItemProps>
```